.gitlab/source_test/ebpf.yml

```diff
@@ -49,7 +49,7 @@
 
 .tests_windows_sysprobe:
   stage: source_test
-  needs: ["go_deps"]
+  needs: ["go_deps", "go_tools_deps"]
```

Comment:
Why do we need this new dependency now?

Reply:
artifact from previous tests removed it 👍